gammapy/maps/region/geom.py

```diff
@@ -782,9 +782,6 @@ def from_hdulist(cls, hdulist, format="ogip", hdu=None):
             regions = []
 
             for region_pix in regions_pix:
-                # TODO: remove workaround once regions issue with fits serialization is sorted out
-                # see https://github.com/astropy/regions/issues/400
-                region_pix.meta["include"] = True
```

Comment:
I think this requires to update regions to >=v0.6. Currently we only require >=v0.5.